website/src/content/development/notifications.mdx

```diff
@@ -0,0 +1,387 @@
+
+<Info>
+
+We recommend displaying `success` notifications in the `side` domain.<br />
+
+</Info>
+
+![Side notifications](/images/notifications-side.png)
+
+# Using notifications in a form
+
+Let's apply notifications in the `Channels` application.
+
+## Form submitted successfully
+
+```js title="channels-details.js" highlightLines="6-7,20,26-30"
+import { useCallback } from 'react';
+import { useRouteMatch } from 'react-router-dom';
+import { useApplicationContext } from '@commercetools-frontend/application-shell-connectors';
+import { FormModalPage } from '@commercetools-frontend/application-components';
+import LoadingSpinner from '@commercetools-uikit/loading-spinner';
+import { useShowNotification } from '@commercetools-frontend/actions-global';
+import { DOMAINS, NOTIFICATION_KINDS_SIDE } from '@commercetools-frontend/constants';
+import useChannelsFetcher from './use-channels-fetcher';
+import useChannelsUpdater from './use-channels-updater';
+import { docToFormValues, formValuesToDoc } from './conversions';
+import ChannelsForm from './channels-form';
+
+const ChannelsDetails = (props) => {
+  const match = useRouteMatch();
+  const languages = useApplicationContext(
+    (context) => context.project.languages
+  );
+  const { data: channel } = useChannelsFetcher(match.params.id);
+  const { updateChannel } = useChannelsUpdater(match.params.id);
+  const showNotification = useShowNotification();
+  const handleSubmit = useCallback(
+    async (formValues) => {
+      const data = formValuesToDoc(formValues);
+      try {
+        await updateChannel(data);
+        showNotification({
+          kind: NOTIFICATION_KINDS_SIDE.success,
+          domain: DOMAINS.SIDE,
+          text: 'Channel updated! 🎉',
+        });
+      } catch (graphQLErrors) {
+        // show an error notification
+      }
+    },
+    [showNotification, updateChannel]
+  );
+
+  if (!channel) {
+    return <LoadingSpinner />;
+  }
+
+  return (
+    <ChannelsForm
+      initialValues={docToFormValues(channel, languages)}
+      onSubmit={handleSubmit}
+    >
+      {(formProps) => {
+        return (
+          <FormModalPage
+            title="Manage Channel"
+            isOpen
+            onClose={props.onClose}
+            isPrimaryButtonDisabled={formProps.isSubmitting}
+            onSecondaryButtonClick={formProps.handleCancel}
+            onPrimaryButtonClick={formProps.submitForm}
+          >
+            {formProps.formElements}
+          </FormModalPage>
+        );
+      }}
+    </ChannelsForm>
+  );
+};
+```
+
+## Form submitted with errors
+
+Most of the time dispatching error notifications is associated with HTTP request handling. In this section we cover handling both GraphQL and REST APIs error on submitting a form.
+
+### GraphQL
+
+<Info>
+
+More information about GraphQL errors can be found on the [GraphQL API page](https://docs.commercetools.com/api/graphql#error-response-format).
```

Comment:
I'm not sure about this link, as this is only true for the CTP API.

How about we link to Apollo docs, as this is what Custom Applications use? https://www.apollographql.com/docs/react/data/error-handling/